src/js/background/background.js

```diff
@@ -111,16 +113,12 @@ async function postReportWebcompatIssue(description) {
     user_agent: description.user_agent
   };
 
-  const newReportedIssueResponse = await fetch(reportWebCompatResponse, {
+  await fetch(reportWebCompatResponse, {
     mode: "same-origin",
     method: "POST",
     headers: headers,
     body: JSON.stringify(apiBody),
   });
```

Comment:
Question (non-blocking) ln 116 in background.js - you have it waiting for fetch to complete (post request) but never return a response regarding if it was successful or not. And on ln 512, you have it just waiting for the outcome of the function, `postReportWebcompatIssue` but it doesn't return anything after it completes the call. We might want to be a little bit more intentional with our code, no? So we know what to expect when we do other calls in the future (response of some sort - not just a side effect that returns silence).

Reply:
Ugh. I think something is wrong with the API from the web-app side. Gonna check with back-end tomorrow.

<img width="514" alt="image" src="https://user-images.githubusercontent.com/2692333/228744106-ad8c8e59-7f55-430c-8de7-229f67c25cfc.png">

Reply:
Per jwhitlock, this is intended. Submitting to this endpoint only returns an HTTP status code. There's no JSON response, etc.